Resources/Prototypes/Entities/Clothing/Neck/cloaks.yml

```
@@ -229,8 +229,8 @@
 - type: entity
   parent: ClothingNeckBase
   id: ClothingNeckCloakBi
-  name: poison cloak
-  description: The purple color is a clear indicator you are poisonous.
+  name: biathalon cloak
```

```suggestion
  name: biathlon cloak
```
I'm pretty sure you don't need the "a" there